### PR TITLE
PMM-T1089 verify ui elements for postgresql

### PR DESCRIPTION
**Pull request by VoK0**

### CODE REVIEW

File: tests/verifyRemoteInstances_test.js [hunk 1]
```diff
@@ -208,6 +208,35 @@ Scenario(
...
+  async ({
+    I, remoteInstancesPage,
+  }) => {
+    I.amOnPage(remoteInstancesPage.url);
+    remoteInstancesPage.waitUntilRemoteInstancesPageLoaded();
+    remoteInstancesPage.openAddRemotePage('postgresql');
+    I.click(remoteInstancesPage.fields.addService);
+    remoteInstancesPage.checkRequiredField();
+    // Verify fields on the page
+    I.waitForVisible(remoteInstancesPage.fields.returnToMenuButton, 30);
```
Comment (puneet0191) [inline tests/verifyRemoteInstances_test.js:222]: waitFor is not wrong, but ideally, I would use I.seeElement, and only use wait for visible when I want to either ensure that the UI component loaded successfully, so here just one WaitForVisible when the Page for adding pgsql opens up and then I.seeElement's

**Review by puneet0191 — APPROVED** [review]
you are faster than Usain Bolt, great work, just a minor note of improvement

**Comment by VoK0** [comment]
https://pmm.cd.percona.com/job/pmm2-ui-tests/19371/allure/#suites/e92ff18239850f86a5e9f72f32894309/5a5d5853b66c8559/

**Review by beatahandzelova — APPROVED** [review]

**Comment by puneet0191** [comment]
Vojtech my friend what are you waiting for, just merge this :+1:

**Pull request merged**